kuma/javascript/src/active-banner.test.js

```diff
@@ -7,11 +7,11 @@ import ActiveBanner, {
 } from './active-banner.jsx';
 import UserProvider from './user-provider.jsx';
 
-const mockUserData = { ...UserProvider.defaultUserData };
-
 describe('ActiveBanner', () => {
+    let mockUserData;
     beforeEach(() => {
         localStorage.clear();
+        mockUserData = { ...UserProvider.defaultUserData };
```

Comment:
Before, the `mockUserData` was created once and mutated across each test making it so that that mutation is applied to each and every test making the tests indeterministic. 

What I *could* do, if you prefer, is to simply put `const mockUserData = { ...UserProvider.defaultUserData };` into every test. Is that neater?